app/src/main/java/com/example/dragonx/view/fragments/RocketDetailsFragment.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.dragonx.view.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModelProvider
+import com.denzcoskun.imageslider.constants.ScaleTypes
+import com.denzcoskun.imageslider.models.SlideModel
+import com.example.dragonx.R
+import com.example.dragonx.models.Rocket
+import com.example.dragonx.viewmodel.RocketDetailsViewModel
+import com.example.dragonx.viewmodel.RocketListViewModel
+import kotlinx.android.synthetic.main.fragment_rocket_details.view.*
+import kotlinx.coroutines.*
+
+class RocketDetailsFragment : Fragment() {
+    private lateinit var viewModel: RocketDetailsViewModel
+    private val rocket = MutableLiveData<Rocket>()
+    private val myJob = Job()
+    private val myScope = CoroutineScope(Dispatchers.IO + myJob)
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_rocket_details, container, false)
+        var rocketNumber = arguments?.getInt("Rocket's number")
```

Comment:
use const value for naming of bundle data

Reply:
Will do